CHANGELOG.md

```diff
@@ -16,7 +16,7 @@ Prefix your items with `(Template)` if the change is about the template and not
 - Enable `TreatWarningsAsErrors` for the Access, Business, and Presentation projects.
 - Update analyzers packages and severity of rules.
 - Fix crash from ARM base mac on net7.0-iOS. Add `ForceSimulatorX64ArchitectureInIDE` property to mobile head.
-- Consolidate build agents to Microsoft-hosted agents.
```

Comment:
For the UnoApplicationTemplate repo, you don't want to use `ci:` when adjusting pipelines because you're actually changing the pipeline of the generated app and not the template itself. You want a new version of the template with those changes but `ci:` doesn't generate new versions. Use `fix` or `chore` instead.

Reply:
You want to use `ci:` when changing things under the .github folder.

Reply:
Noted will switch to `chore`.
Would `build:` be also acceptable for this PR?